js/app.js

```diff
@@ -548,32 +552,73 @@ FaFp+DyAe+b4nDwuJaW2LURbr8AEZga7oQj0uYxcYw==\n\
     }
     var value = getTrimmedValue(tokenEditor);
     //var isBase64 = isBase64EncodedElement.checked;
-    /*
-    if (isBase64 && !window.isValidBase64String(secretElement.value)) {
+    
+    /*if (isBase64 && !window.isValidBase64String(secretElement.value)) {
       $(signatureContainerElement).addClass('error');
       return;
     } else {
       $(signatureContainerElement).removeClass('error');
-    }
-    */
+    }*/
+    
 
     var result = false;
     if (algorithm == "RS256") {
+		//Get the pasted token
+		newToken = tokenEditor.getValue();
+		//newToken = "blah";
+		
+      $.ajax({
```

Comment:
Really odd spacing.  Are these tabs?  I think JS is usually 2 or 4 spaces.  The rest of the file tabbing is 2 spaces.  Please update!

Reply:
Updated.

Reply:
Still has odd tabbing.